cmd/pocketd/cmd/config.go

```diff
@@ -6,7 +6,7 @@ import (
 	"pocket/app"
 )
 
-func initSDKConfig() {
+func InitSDKConfig() {
```

Comment:
If this is only being exposed for testing purposes, wdyt about adding a  `test_config.go`  file which exposes a wrapper and has a `//go:build test` tag/constraint?

```go
//go:build test

package cmd

// InitSDKConfig exports initAppConfig for use in tests
func InitSDKConfig() {
	initSDKConfig()
}
```

(see: https://github.com/pokt-network/poktroll/pull/42/files#diff-fe3797aa7d51913c0f4b02574c878cf02042fb8a159d5355a2c1b3731490b982R1)

We would have to update the makefile as well:

```makefile
.PHONY: go_test
go_test: go_version_check ## Run all go tests
	go test -v -race -tags test ./...

.PHONY: go_test_integration
go_test_integration: go_version_check ## Run all go tests
	go test -v -race -tags test,integration ./...
```

(see: https://github.com/pokt-network/poktroll/pull/42/files#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52R94-R98)

Reply:
Previously we had discussed adding `//go:build test` to all test files to mitigate this concern. The user experience would be that without the `-tags test` constraint, the `go test` would say that "no tests were run".

Reply:
I added a TODO link to this discussion. Is it alright if follow up on it outside of this PR?

If we do go with this approach, we should have really good CLI communication to the user to make it obvious how necessary the tag is like `YOU HAVE NOT ADDED THIS TAG SO NO TESTS ARE BEING EXECUTED`.

Reply:
For sure, we can have that discussion elsewhere. I'm just now recalling that had also done what you're suggesting :joy:. IIRC, there was a test which did not have the `test` build constraint and would check whether the `-tags` flag contained `test`, and return a similar error if not. I'll see if I can find it but otherwise, it should be straightforward to reproduce.